dev-docs/bidders/between.md

```diff
@@ -5,7 +5,9 @@ description: Prebid Between Bidder Adaptor
 pbjs: true
 pbs: true
 biddercode: between
-media_types: banner
+schain_supported: true
```

Comment:
I don't see your bid adapter looking for bidRequest.schain -- https://github.com/prebid/Prebid.js/blob/master/modules/betweenBidAdapter.js -- is there a PR open?

Reply:
It's going in soon, I believe.

cc @ignat-one